### DISCUSSION TITLE
Server has the same photo message

### DISCUSSION THREAD

**Discussion by jedsta** [simulot/immich-go#252] [Q&A]
I have two files on the server, ABC123.**jpg** which is 60k and ABC123 (1).**JPG** which is 1.2Mb. These are the same image, just with different names, dates and quality.

I have a local version of the the file ABC123.**JPG** which is also 1.2Mb. When I run immich-go It logs that the file already exists on the server and skips it.

Is there a way to have it identify that the local file is a larger size / higher quality and replace the server version? Or does it somehow know the file with (1) is the same file with a different name?

Also, the capitalization on the filename extension I am mentioning in the example are not matching in case, which I mention in case it is relevant to this happening.

**Comment by simulot** [comment]
immich-go compares size of files, and keep the heaviest. then when the local files is heavier than the server's file, the local file replace the server's version. Album membership is kept.
